cudos/chain.json

```diff
@@ -164,6 +168,10 @@
       {
         "address": "https://cudos-api.lavenderfive.com/",
         "provider": "Lavender.Five Nodes 🐝"
+      },
+      {
+        "address": "https://celestia-rest.publicnode.com",
```

Comment:
this says celestia but it's in cudos' chain.json

Reply:
Thank you. Fixed